### PR TITLE
Enable service proxy in vlan bridge mode

### PR DESCRIPTION
**Pull request by jojimt**
This allows the proxy to be used with ACI

### CODE REVIEW

**Comment by shaleman** [comment]
Don't we need to change the dest-mac based on which subnet client and providers are?

**Comment by jojimt** [comment]
I have opened an issue to track that item. I will add it as a separate PR.

File: vlanBridge.go [hunk 1]
```diff
@@ -89,6 +92,11 @@ func (vl *VlanBridge) SwitchDisconnected(sw *ofctrl.OFSwitch) {
 
 // PacketRcvd Handle incoming packet
 func (vl *VlanBridge) PacketRcvd(sw *ofctrl.OFSwitch, pkt *ofctrl.PacketIn) {
+	if pkt.TableId == SRV_PROXY_SNAT_TBL_ID || pkt.TableId == SRV_PROXY_DNAT_TBL_ID {
+		// these are destined to service proxy
+		vl.svcProxy.HandlePkt(pkt)
+	}
```
Comment (shaleman) [inline vlanBridge.go]: Should we return from here since rest of the handling below doesn't apply to packets destined to service proxy?

**Comment by shaleman** [comment]
Sounds good @jojimt 
LGTM other than the minor comment

**Pull request merged**